front/components/assistant/AssistantPicker.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Button,
+  DropdownMenu,
+  MagnifyingGlassStrokeIcon,
+  PlusIcon,
+  RobotIcon,
+  WrenchIcon,
+} from "@dust-tt/sparkle";
+import Link from "next/link";
+import { useState } from "react";
+
+import { AgentConfigurationType } from "@app/types/assistant/agent";
+import { WorkspaceType } from "@app/types/user";
+
+export function AssistantPicker({
+  owner,
+  assistants,
+  onItemClick,
+  pickerButton,
+  manageButtons,
+}: {
+  owner: WorkspaceType;
+  assistants: AgentConfigurationType[];
+  onItemClick: (assistant: AgentConfigurationType) => void;
+  pickerButton?: React.ReactNode;
+  manageButtons?: boolean;
+}) {
+  const [searchText, setSearchText] = useState("");
+  const searchedAssistants = assistants.filter((a) =>
+    a.name.toLowerCase().startsWith(searchText.toLowerCase())
+  );
+  return (
+    <DropdownMenu>
+      <div onClick={() => setSearchText("")}>
+        {pickerButton ? (
+          <DropdownMenu.Button>{pickerButton}</DropdownMenu.Button>
+        ) : (
+          <DropdownMenu.Button icon={RobotIcon} />
+        )}
+      </div>
+      <DropdownMenu.Items origin="auto" width={240}>
+        {assistants.length > 7 && (
+          <div className="border-b border-structure-100 p-2">
+            <div className="relative text-sm font-medium text-element-800">
```

Comment:
I think we want a sparkle component here. There are a few things that are not quite aligned with the sparkle Input component and our design system afaict. In particular the outline/focus color is wrong.

Let's create a search component (will forward you francesco diff in case useful) in sparkle have it reviewed by @Duncid and then use it here.

Reply:
Yep we had that discussion with @Duncid yesterday and I agreed, will do

however, I had specifically asked @Duncid about the outline / focus color  and he said that part was fine